src/components/CardInput/CardOwnerNameInput.tsx

```diff
@@ -0,0 +1,52 @@
+import { ChangeEvent, forwardRef, KeyboardEvent, RefObject } from 'react';
+import { CARD_INPUT_COLOR, CARD_INPUT_TEXT_FONT_SIZE, CARD_INPUT_TEXT_FONT_WEIGHT } from './constants/cardInputStyles';
+import { HStack, Label, TextField, Typography, VStack } from '@/shared/components';
+import { styleToken } from '@/shared/styles';
+
+const OWNER_NAME_INPUT_MAX_VALUE_LENGTH = 30;
+const OWNER_NAME_INPUT_ID = 'owner-name-input';
+const OWNER_NAME_INPUT_PLACEHOLDER = '카드에 표시된 이름과 동일하게 입력하세요.';
+
+type OwnerNameInputProps = Partial<{
+  ref: RefObject<HTMLInputElement>;
+  value: string;
+  onChange: (event: ChangeEvent<HTMLInputElement>) => void;
+  onKeyUp: (event: KeyboardEvent<HTMLInputElement>) => void;
+}>;
+
+export const CardOwnerNameInput = forwardRef<HTMLInputElement, OwnerNameInputProps>(
+  ({ ref, value, onChange, onKeyUp }: OwnerNameInputProps) => (
```

Comment:
```suggestion
type OwnerNameInputProps = Partial<{
  value: string;
  onChange: (event: ChangeEvent<HTMLInputElement>) => void;
  onKeyUp: (event: KeyboardEvent<HTMLInputElement>) => void;
}>;

export const CardOwnerNameInput = forwardRef<HTMLInputElement, OwnerNameInputProps>(
  ({ value, onChange, onKeyUp }, ref) => (
```
이렇게 표현되어야 할 것 같네요!

https://react.dev/reference/react/forwardRef#render-function

<img width="932" alt="image" src="https://github.com/next-step/react-payments/assets/18749057/54b02479-eecf-42f6-980d-870a097145d9">

Reply:
> 이렇게 표현되어야 할 것 같네요!

forwardRef 구조로 컴포넌트를 변경하면서 타입을 제대로 신경쓰지 못했네요!
개선 할 점 봐주셔서 감사합니다! ☺️